.github/workflows/reusable-e2e.yml

```diff
@@ -239,7 +238,7 @@ jobs:
 
       - name: Install packages
         if: steps.restore-cache.outputs.cache-hit != 'true' || steps.restore-cypress-cache.outputs.cache-hit != 'true'
-        run: yarn --no-lockfile
+        uses: ./.github/actions/install-with-retries
```

Comment:
why would the retries help here?

Reply:
It won't help our specific problem, but I just added it here as a minor improvement.